### ISSUE TITLE
Como alguém deve reportar uma violação

### ISSUE THREAD

**Issue by felipenmoura** [NiteroiJS/code-of-conduct#7]
Muitas vezes a pessoa vê algo que acredita ser uma violação e não sabe como reporta-la, ou se manifesta de formas, muitas vezes inapropriadas, como em redes sociais, antes mesmo da própria organização do evento ter conhecimento do assunto!

**Comment by leobalter** [comment]
o ideal seria ter pelo menos 2 formas para contato: um email específico para falar sobre o código de conduta e um formulário de contato, se esse código ficar em algum site.

O formulario permite um anonimato.

Para esses 2 canais é bom deixar certo de quem vai receber essa mensagem (um comite do grupo para avaliar?) e que as informações pessoas serao preservadas, principalmente de quem estaria violando.

Talvez isso faça parte de outra issue, mas é bom que uma denuncia, nao seja exposta inclusive para o acusado de violação, evitando que o problema seja aumentado. O que for falado com o violador tem que separar apenas as informações necessárias.

**Comment by felipenmoura** [comment]
exato, é importante uma "discrição" e anonimato, para não expor nem a vítima nem o transgressor, ou qualquer outro indivíduo.
Um e-mail específico para isto é complicado, envolveria até mesmo custos, etc...mas um formulário, é uma boa, sim, e uma forma rápida e simples tanto para divulgar quanto reportar.

**Comment by leobalter** [comment]
Um formulário manda os dados para um e-mail, um email é só um endereço. A ideia desse código é focar em um grupo com a ideia que ele terá fork para detalhes ajustados para outros grupos e eventos. Logo, posso dizer pelo NiteroiJS e RioJS que um e-mail não seria um custo. 

Mais que a parte de custos. A gente pode falar sobre o que seria o ideal, vamos ver quais as formas eficazes de comunicação, mesmo que elas sejam custosas, qual seria a melhor para uma vítima reportar um problema.

**Comment by leobetosouza** [comment]
O bom do form é q ele pode ser criado com remetente opcional, assim a
pessoa q ta fazendo a denuncia pode optar por ficar anonimo.

E conta de email gratuita é o q mais tem. Mas quem teria acesso ao inbox ou
à planilha de denuncias? Os moderadores/organizadores?

E se o denunciado for um desses ultimos?

---

_Leonardo Alberto A. de J. Souza - Leobetosouza_

2015-08-28 0:17 GMT-03:00 Leo Balter notifications@github.com:

**Comment by leobalter** [comment]
Ambos têm vantagens e desvantagens, por isso uma ideia de ter mais de um canal
